homework2/task1.py

```diff
@@ -0,0 +1,8 @@
+# Напишите программу, которая принимает на вход вещественное число и показывает сумму его цифр.
+
+a = input('введите число: ')
+sum = 0
+for i in a:
+    if i != '.' and i != ',' and i.isnumeric():
```

Comment:
```suggestion
    if i.isnumeric():
```
Тут хватит и одного условия. Ну усложняйте.